website/snippets/cloud-feature-parity.md

```diff
@@ -6,7 +6,7 @@ The following table outlines which dbt Cloud features are supported on the diffe
 | dbt Cloud IDE                 | ✅           | ✅                     | ✅                  |
 | dbt Cloud CLI                 | ✅           | ✅                     | ❌                  |
 | Audit logs                    | ✅           | ✅                     | ✅                  |  
-| Discovery API                 | ✅           | ✅                     | ❌                  | 
+| Discovery API                 | ✅           | ✅                     | ✅                  | 
 | dbt Explorer                  | ✅           | ✅                     | ❌                  | 
```

Comment:
is dbt explorer also available in azure st? @nghi-ly mentioned that @runleonarun was looking into it: https://dbt-labs.slack.com/archives/C04559AB7D2/p1714489073253449?thread_ts=1714488772.813649&cid=C04559AB7D2